text/0000-reseeding-rand.md

```diff
@@ -0,0 +1,682 @@
+- Start Date: 2013-01-23
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Stabilise the `std::rand` module by focusing it on a smaller set of
+functionality, moving "fancy" functionality to a crates.io crate.
+
+# Motivation
+
+The `std::rand` module provides various facilities for working with
+random numbers and randomness in general. This is a surprisingly
+sensitive topic, with different use cases. `std` does not aim to
+completely satisfy everything, but rather aims to provide a minimal
+set of functionality that tries to be simple and straightforward
+without restricting people building on it, and this reasoning extends
+to `std::rand`.
+
+A major area of use for randomness is cryptography where the
+recommended approach is to use the operating system if at all
+possible. There is no particular reason for `std::rand` to deviate
+from this. However, other uses of randomness (e.g. simulations) do not
```

Comment:
Performance matters for most uses of a CSPRNG. There is a *reason* that so much effort goes into creating faster stream ciphers.